sdk/README.md

```diff
@@ -203,13 +203,8 @@ your code changes are improving the number of successfully compiled KFP pipeline
 ## Troubleshooting
 
 - When you encounter ServiceAccount related permission issues, refer to the
-  ["Servince Account and RBAC" doc](sa-and-rbac.md)
+  ["Service Account and RBAC" doc](sa-and-rbac.md)
   
 - If you run into the error `bad interpreter: No such file or director` when trying 
   to use Python's venv, remove the current virtual environment in the `.venv` directory
   and create a new one using `virtualenv .venv`
-
-- For big data passing, PVs must be created manually, or dynamic volume provisioning,
-  has to be enabled (https://kubernetes.io/docs/concepts/storage/dynamic-provisioning).
-  The PVC name has to match the PipelineRun name until
-  [issue #181](https://github.com/kubeflow/kfp-tekton/issues/181) has been addressed.
```

Comment:
@fenglixa @Tomcli -- I removed this since issue #181 was resolved by PR #309

Reply:
Thanks, this is fixed in #309